Makefile

```diff
@@ -11,3 +11,10 @@ deps:
 
 test:
 	go test ./...
+
+integration-test:
+	docker-compose -f Dockercompose.test.yml up --build --abort-on-container-exit --always-recreate-deps
+	docker-compose -f Dockercompose.test.yml down --volumes
```

Comment:
you could replace these with a `docker-compose run golang_tests` so it automatically stops after running them